packages/core/src/prompts/lesson-assistant/parts/interactive.ts

```diff
@@ -1,12 +1,22 @@
 import { TemplateProps } from "..";
 
+const STRUCTURED_OUTPUTS_ENABLED =
+  process.env.NEXT_PUBLIC_STRUCTURED_OUTPUTS_ENABLED === "true" ? true : false;
```

Comment:
I'm not so happy about this, because we aren't adding another prompt variant. But that felt like too big a change for just now.

Note that if we're not using structured outputs we need to re-add the language about sending back JSON patch documents for each change rather than a single object with patches and a user prompt.